google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Subscriber.java

```diff
@@ -77,7 +75,7 @@
  * <p>If no credentials are provided, the {@link Subscriber} will use application default
  * credentials through {@link GoogleCredentials#getApplicationDefault}.
  */
-public class Subscriber {
+public class Subscriber extends AbstractApiService {
```

Comment:
I didn't look over this file very carefully - am I correct in assuming that this is mostly a revert to the version that extended Guava's `AbstractService`?